### ISSUE TITLE
Tizen 3 Sintel 4K stream error

### ISSUE THREAD

**Issue by laczkog** [google/shaka-player#2842]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**What version of Shaka Player are you using?**
2.5.14

**Can you reproduce the issue with our latest release version?**
Yes

**Can you reproduce the issue with the latest code from `master`?**
Yes

**Are you using the demo app or your own custom app?**
Custom

**If custom app, can you reproduce the issue using our demo app?**
Yes

**What browser and OS are you using?**
Tizen 3 (Samsung 2017)

**For embedded devices (smart TVs, etc.), what model and firmware version are you using?**
UE55MU6122, firmware version:180

**What are the manifest and license server URIs?**
manifestUri: 'https://storage.googleapis.com/shaka-demo-assets/sintel-widevine/dash.mpd'
com.widevine.alpha: 'https://cwip-shaka-proxy.appspot.com/no_auth',

**What did you do?**
Loaded and started the stream.

**What did you expect to happen?**
Stream goes up to uhd resolution, because the TV is capable of uhd, and there is enough bandwidth.

**What actually happened?**
Error when the stream changed to uhd resolution.

**Comment by joeyparrish** [comment]
> **What actually happened?**
> Error when the stream changed to uhd resolution.

Can you please tell us what the error was?

**Comment by laczkog** [comment]
Sorry, I forgot that.

The player did not throw an error, but the video did, which was error code 3 (VIDEO_DECODE_ERROR).

I read it's out of control of javascript, but with Tizen, I can't get to chrome://media-internals to get more information.

**Comment by joeyparrish** [comment]
I don't know for certain, but I believe Tizen 3 is WebKit-based, and so wouldn't have chrome:// URLs.

**Comment by laczkog** [comment]
https://developer.samsung.com/smarttv/develop/specifications/web-engine-specifications.html

According to this website, from 2017, Tizen is based on chromium.

**Comment by joeyparrish** [comment]
I stand corrected!  But it's not a given that Samsung would enable those features in their build, so it's still not super surprising that chrome:media-internals doesn't work on Tizen 3.

We have a Tizen 3 device in our lab, so we should be able to replicate this.  We may not be able to get much more info, but we'll just have to try and see what we can see.

Thanks for providing all the necessary details in your report!

**Comment by joeyparrish** [comment]
I've discovered something surprising.

First, I played the content on Tizen, using an automated playback test that forces low res.  It played fine.

Then, I did it again, but modified the config to force 4k.  It failed, as you reported.  Same results with the 1440p track.

When I played this same content on Chrome, I noticed that there was **no 4k track** in the UI at all.  Same for 1440p.

I did some more digging and discovered the same thing on both Chrome and Tizen: we don't have UHD keys in the license.

I'll work to fix this in the license proxy and see if that fixes playback.
